Scripts/Source/UD_MCM_script.psc

```diff
@@ -2218,6 +2227,8 @@ Function DDPatchPageDefault(int option)
         SetInfoText("Prevent NPC outfit from being removed when hand restraint is locked on. Removing outfit can by default cause compatibility issue with NPC overhaul mods. This will obviously prevent NPC from being naked until player undresses them\nDefault: True")
     elseif option == UD_OrgasmAnimation_M
         SetInfoText("List of orgasm animations.\nNormal = Normal orgasm animations by  DD\nExtended = Orgasm animations + horny animations\nDefault: Normal")
+    elseif option == UD_AllowMenBondage_T
```

Comment:
There is no need to add this part, as the whole Default part is still WIP. Because of that, I kept the previous info texts as template for future use (so there is no need to write all the ifs again). Will have to finish it myself or create request

Reply:
Well, this doesn't bad, right? And will be useful later to make sure this is not forgotten, right?

Reply:
You are right. Sorry about that.